docs/users/cli-usage.md

```diff
@@ -192,7 +192,7 @@ Note that the git URL shown below is an example only, you'll need to use your ow
 ```bash
 git clone https://github.com/example/example-site
 cd example-site
-ddev config --auto
+ddev config --project-type=drupal9
```

Comment:
```suggestion
ddev config --project-type=drupal9 --docroot=web --create-docroot
```

This resolves the problem that the comment was talking about. I never actually knew that the docroot might not exist in a Drupal9 project.